web/src/app/main/App.tsx

```diff
@@ -54,6 +56,9 @@ const useStyles = makeStyles((theme: Theme) => ({
 }))
 
 export default function App(): JSX.Element {
+  const [analyticsID] = useConfigValue('General.GoogleAnalyticsID') as [string]
+  if (analyticsID) ReactGA.initialize(analyticsID)
```

Comment:
should this be in a useEffect so it doesn't get called every render, or is it safe here?

Reply:
yea that's a good point, I'll throw that in a useEffect